packages/bottom-tabs/src/views/BottomTabBar.tsx

```diff
@@ -249,18 +260,87 @@ export default function BottomTabBar({
     descriptors,
     dimensions,
     layout,
+    tabBarPosition,
   });
 
   const tabBarBackgroundElement = tabBarBackground?.();
 
+  const tabBarPositionMap: TabBarPositionMapConfig = {
+    right: {
+      tabBarItem: {
+        height:
+          Platform.OS === ('ios' || 'android')
+            ? tabBarHeight / 1.5
+            : DEFAULT_TABBAR_HEIGHT,
+        marginTop: COMPACT_TABBAR_HEIGHT / 2,
+        flex: Platform.OS === ('ios' || 'android') ? 0 : 0.07,
+      },
+      tabBar: {
+        paddingHorizontal: 5,
+        borderLeftColor: colors.border,
+        paddingTop: insets.top + tabBarHeight,
+        borderLeftWidth: StyleSheet.hairlineWidth,
+      },
+      content: {
+        flex: 1,
+        width: tabBarHeight,
+        maxWidth:
+          Platform.OS === ('ios' || 'android')
+            ? DEFAULT_TABBAR_HEIGHT
+            : DEFAULT_TABBAR_HEIGHT * 1.3,
+      },
+    },
+
+    left: {
+      tabBarItem: {
+        height:
+          Platform.OS === ('ios' || 'android')
+            ? tabBarHeight / 1.5
+            : DEFAULT_TABBAR_HEIGHT,
+        marginTop: COMPACT_TABBAR_HEIGHT / 2,
+        flex: Platform.OS === ('ios' || 'android') ? 0 : 0.07,
+      },
+      tabBar: {
+        paddingHorizontal: 5,
+        borderRightColor: colors.border,
+        paddingTop: insets.top + tabBarHeight,
+        borderRightWidth: StyleSheet.hairlineWidth,
+      },
+      content: {
+        flex: 1,
+        width: tabBarHeight,
+        maxWidth:
+          Platform.OS === ('ios' || 'android')
+            ? DEFAULT_TABBAR_HEIGHT
+            : DEFAULT_TABBAR_HEIGHT * 1.3,
+      },
+    },
```

Comment:
This code is almost the same as the `right` style apart from the `borderRightColor` and `borderRightWidth`. We should define those 2 conditionally instead of duplicating the whole style.

Reply:
Got you, this has been fixed with a different approach